docs/storage-abstraction.md

```diff
@@ -47,157 +69,58 @@ You can create a JSON file emoney-storage.json with the JSON below.
 }
 ```
 
-Note that the `transaction` field is set to `false`, which indicates you use this table with the Storage API.
+To apply the schema, go to the [`scalardb Releases`](https://github.com/scalar-labs/scalardb/releases) page and download the ScalarDB Schema Loader that matches the version of ScalarDB that you are using to the `getting-started` folder.
 
-To apply the schema, download the Schema Loader that matches the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
+Then, run the following command, replacing `<VERSION>` with the version of the ScalarDB Schema Loader that you downloaded:
```

Comment:
> replacing `<VERSION>` with

[minor] The following command uses lower-case `<version>`. We should use the same style?

Reply:
Good catch! I've changed `<version>` to `<VERSION>`.